test/integration/helpers/fileExists.js

```diff
@@ -11,6 +11,9 @@ const open = require('fs/promises').open;
 */
 module.exports = function fileExists(file) {
```

Comment:
We could use `fs.access` here instead, which doesn't open the file https://nodejs.org/api/fs.html#fs_fspromises_access_path_mode

Reply:
Nice suggestion 🙂 I've updated the pull-request to use fs.access instead 👍